### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by ritikaawasthy**

### CODE REVIEW

File: js/homepage.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
+
+  let imgPath="/media/home/";
```
Comment (Debarshi95) [inline js/homepage.js:2]: If imgPath is not re-assigned later in the code, using ```const``` would be better here.

**Review by Debarshi95 — APPROVED** [review]
Looks good.

File: css/color.css [hunk 2]
```diff
@@ -0,0 +1,43 @@
+html{
+  --primary: #65012E;
+  --secondary:  #30343F;
+  --background:  #F1EAE4;
```
Comment (manojsarna) [inline css/color.css:4]: define color codes in root

File: js/homepage.js (hunk 1 above)
Comment (manojsarna) [inline js/homepage.js:2]: use const

File: js/homepage.js [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+  let slider1Fwd= document.getElementById('slid-t-btn-fwd');
+  let slider2Fwd= document.getElementById('slid-m-btn-fwd');
+  let slider3Fwd= document.getElementById('slid-b-btn-fwd');
+  let slider1Rev= document.getElementById('slid-t-btn-rev');
+  let slider2Rev= document.getElementById('slid-m-btn-rev');
+  let slider3Rev= document.getElementById('slid-b-btn-rev');
+  let sideMenuBtn= document.getElementById('side-menu-btn');
+  let sideMenu= document.getElementById('side-menu');
+let currentPg= 1;
+let lastPage=3;
```
Comment (manojsarna) [inline js/homepage.js:12]: can be const

File: templates/home.html [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+        <li ><i class="fas fa-heart"></i></li>
+        <li ><i class="fas fa-shopping-cart"></i></li>
+      </ul>
+    </nav>
+  </header>
+
+
+  <ul class="list-container list-stacked hide" id="side-menu" >
+    <li class="fw-b" >Women</li>
+    <li>
```
Comment (manojsarna) [inline templates/home.html:36]: check spacing

File: templates/home.html [hunk 5]
```diff
@@ -0,0 +1,186 @@
...
+
+        </div>
+      </article>
+    </section>
+</section>
+
+
+<section class="slider-bottom">
+  <h2 class="center-txt fw-reg primary-col">Footwear</h2>
+    <section class="grid-two-layout-container">
```
Comment (manojsarna) [inline templates/home.html:119]: here too

File: templates/home.html [hunk 6]
```diff
@@ -0,0 +1,186 @@
...
+          <p>Industrial Textile</p>
+          <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, </p>
+          <p>sed do eiusmod tempor incididunt ut labore sunt in culpa </p>
+          <p>qui officia deserunt mollit anim id est laborum.</p>
+        </div>
+
+      </article>
+
+      <article class="grid-three-layout-center landing-footer-section">
+        <div class="">
```
Comment (manojsarna) [inline templates/home.html:162]: keep consistent spacing

**Review by manojsarna — APPROVED** [review]
UI Review : Superb!!

Suggestions : add some cards, bestsellers on homepage , favorite categories

**Pull request merged**